assets/css/app.css

```diff
@@ -47,3 +47,22 @@
   filter: invert(17%) sepia(81%) saturate(3321%) hue-rotate(352deg)
     brightness(96%) contrast(99%);
 }
+
+
+@keyframes clickAnimation {
+  0% {background-color: #001A72}
+  100% {background-color:rgba(255, 255, 255, 0.4);}
+}
+
+.animate {
+  animation: clickAnimation 0.2s ease-in-out;
+}
+
+input,
+textarea,
+button,
+select,
+a {
+  -webkit-tap-highlight-color: transparent;
```

Comment:
I'm not sure removing the highlight using a non-standard attribute is a proper solution. Why is the tooltip part of the clickable link and thus highlighted? Maybe tweaking that would be better?

Reply:
You are right, I've removed it from this PR and I will focus on it on the second one related to TAB navigation.